web_s2i/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:18.04
+
+COPY . /
+
+RUN apt-get update && \
+    apt-get install -y python3-pip libmysqlclient-dev && \
+    pip3 install -r requirements.txt
+
+ENTRYPOINT ["./application.py"]
```

Comment:
Any special reason the application is the entrypoint and not cmd?

Reply:
I used ENTRYPOINT since it was already there (took @lioramilbaum code), no special reason. What is recommended?

Reply:
`ENTRYPOINT` is mostly used as the `shell` of the container, if the application accepts parameters (like here: https://github.com/hashicorp/docker-consul/blob/c5cfe2b2554acdbd06add9e71373b65d167b1ade/0.X/Dockerfile) then the `ENTRYPOINT` it's all a question of what part is the command itself, and what part is it's arguments.
Since the application itself doesn't get parameters and not intended to run like that, then it seems to me that using 
`ENTRYPOINT ["python3"]`
and
`CMD ["./application.py"]`
Will also allow us to run the tests more easily with the container if we'd want it

Reply:
done